Views/Players/PlayerInfo.cshtml

```diff
@@ -1,17 +1,15 @@
 ﻿@using dotnetNewProject.Models
-@model Player
-
+@model List<Player>
 <ul>
-    @foreach (var player in ViewData["players"] as IList<Player>)
+    @foreach (Player player in Model)

```

Comment:
What: Where possible, use `var` to declare variables, don't have to write it twice
Why: It's just more terse
When: When the type can be inferred during assignment.

```diff
- ModifiedPersonComponentTree tree = new ModifiedPersonComponentTree ()
- String personName = "Claribel";
- DateTime personDate = new date();
- bool personIsNice = true;
+ var tree = new ModifiedPersonComponentTree ()
+ var personName = "Claribel";
+ var personDate = new date();
+ var personIsNice = true;
```